.gitattributes

```diff
@@ -2,5 +2,4 @@
 /test               export-ignore
 /.gitattributes     export-ignore
 /.gitignore         export-ignore
-/.travis.yml        export-ignore
```

Comment:
Please add `.github` instead

Reply:
Thanks for your review. I will keep my PR as is.

My intention is to delete a thing that no longer exist.
When the file is cleaned up, it can be improved further.

Reply:
You can click apply on my suggestion
that will make your pr more accurate

Reply:
You can create your own PR if you want.

As I've said, my intention is different. Thank you.

Reply:
done in https://github.com/erusev/parsedown/pull/905